postgrestest.go

```diff
@@ -62,18 +69,27 @@ func New() (*DB, error) {
 	}, nil
 }
 
-// Close closes the database connection and removes
-// the test database.
+// Close removes the test database and closes the database connection. This
+// method should not be called from multiple goroutines.
 func (pg *DB) Close() error {
 	if os.Getenv("PGTESTKEEPDB") != "" {
 		fmt.Fprintf(os.Stderr, "postgrestest schema: %v\n", pg.schema)
 		fmt.Fprintf(os.Stderr, "\tSET search_path TO %q;\n", pg.schema)
 		fmt.Fprintf(os.Stderr, "\tDROP SCHEMA %q CASCADE;\n", pg.schema)
 		return nil
 	}
-	// Drop the schema in a goroutine so that if it fails because some goroutine is maintaining
-	// a lock on a table, we can time out instead of hanging up indefinitely
+
+	// If for some reason someone replaced our DB with nil, there's nothing to
```

Comment:
Might return early on pg.DB == nil as the first thing in this method. The Fprintfs above are harmless, but it might be confusing to see them if the db was nil?